### PR TITLE
I've added 2 static memners & 1 protected,created a new function=it's…

### PR DESCRIPTION
**Pull request by MartinElizbaryan**
…  return everything of Profile

### CODE REVIEW

File: src/Project/Man.php [hunk 1]
```diff
@@ -23,6 +23,22 @@ class Man
      */
     private $birthDate;
 
+    /**
+     * @var int
+     */
+    public static $handsCount = 2;
```
Comment (MartinElizbaryan) [inline src/Project/Man.php]: This is static because everyone's got 2 hands

Comment (azatyan) [inline src/Project/Man.php]: apres. good for first day of coding. 👍  will merge. you are free today.

File: src/Project/Man.php [hunk 2]
```diff
@@ -58,4 +74,23 @@ public function getFullName(){
         
         return $fullName;
     }
-}
\ No newline at end of file
+
+
+    /*
+     * Get the Man's Full Name
```
Comment (azatyan) [inline src/Project/Man.php]: @MartinElizbaryan  fix description

File: src/Project/Man.php [hunk 3]
```diff
@@ -58,4 +74,23 @@ public function getFullName(){
...
+     * @return string
+     *
+     */
+    public function getProfile(){
+
+        return [
+            'firstName' => $this->firstName,
```
Comment (azatyan) [inline src/Project/Man.php]: @MartinElizbaryan  lets use lower case with dashes for all Keys (only keys)

**Comment by azatyan** [comment]
Congratulations for first PR in your coding life.

**Comment by MartinElizbaryan** [comment]
Skype [ I've done ]


but after nano clone { and the link} , I pushed ctrl O than coudn't
 push ctrl X......it wasn't there....why??

**Comment by azatyan** [comment]
Call me tomorrow morning at 11:00 am, i will help

**Pull request merged**